modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/ProductResourceTest.java

```diff
@@ -50,6 +66,16 @@ public void setUp() throws Exception {
 			testGroup.getGroupId(), RandomTestUtil.randomString());
 	}
 
+	@Override
+	@Test
+	public void testGetChannelProductsPageWithFilterStringEquals()
+		throws Exception {
+
+		super.testGetChannelProductsPageWithFilterStringEquals();
+
+		_testGetChannelProductsPageWithFilterStringEqualsCustomFields();
```

Comment:
can we move this in the _testGetChannelProductsPage test? and rename it according